appinfo/routes.php

```diff
@@ -46,110 +46,111 @@
 	],
 	'ocs' => [
 		[
-			'name' => 'api#getRooms',
-			'url' => '/api/{apiVersion}/room',
-			'verb' => 'GET',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#makePublic',
-			'url' => '/api/{apiVersion}/room/public',
-			'verb' => 'POST',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#makePrivate',
-			'url' => '/api/{apiVersion}/room/public',
-			'verb' => 'DELETE',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#getRoom',
-			'url' => '/api/{apiVersion}/room/{token}',
+			'name' => 'Call#getPeersForCall',
+			'url' => '/api/{apiVersion}/room/{token}/peers',
 			'verb' => 'GET',
 			'requirements' => [
 				'apiVersion' => 'v1',
 				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
 		[
-			'name' => 'api#renameRoom',
-			'url' => '/api/{apiVersion}/room/{roomId}',
-			'verb' => 'PUT',
+			'name' => 'Call#joinCall',
+			'url' => '/api/{apiVersion}/room/{token}/join',
+			'verb' => 'POST',
 			'requirements' => [
 				'apiVersion' => 'v1',
-				'roomId' => '\d+'
+				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
 		[
-			'name' => 'api#addParticipantToRoom',
-			'url' => '/api/{apiVersion}/room/{roomId}',
+			'name' => 'Call#pingCall',
+			'url' => '/api/{apiVersion}/ping',
 			'verb' => 'POST',
 			'requirements' => [
 				'apiVersion' => 'v1',
-				'roomId' => '\d+'
+				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
 		[
-			'name' => 'api#leaveRoom',
-			'url' => '/api/{apiVersion}/room/{roomId}',
+			'name' => 'Call#leaveCall',
+			'url' => '/api/{apiVersion}/leave',
 			'verb' => 'DELETE',
-			'requirements' => [
-				'apiVersion' => 'v1',
-				'roomId' => '\d+'
-			],
+			'requirements' => ['apiVersion' => 'v1'],
 		],
+
 		[
-			'name' => 'api#getPeersInRoom',
-			'url' => '/api/{apiVersion}/room/{token}/peers',
+			'name' => 'Room#getRooms',
+			'url' => '/api/{apiVersion}/room',
 			'verb' => 'GET',
-			'requirements' => [
-				'apiVersion' => 'v1',
-				'token' => '^[a-z0-9]{4,30}$',
-			],
+			'requirements' => ['apiVersion' => 'v1'],
 		],
 		[
-			'name' => 'api#joinRoom',
-			'url' => '/api/{apiVersion}/room/{token}/join',
-			'verb' => 'POST',
+			'name' => 'Room#getRoom',
+			'url' => '/api/{apiVersion}/room/{token}',
```

Comment:
What is `token` here, the call token? If so, shouldn't this be in the call controller? If it's the room id, could you please rename it to avoid confusion with `token` as used above (which is a call token)?